nodebuilder/tests/api_test.go

```diff
@@ -13,11 +13,27 @@ import (
 	"github.com/celestiaorg/celestia-node/api/rpc/client"
 	"github.com/celestiaorg/celestia-node/blob"
 	"github.com/celestiaorg/celestia-node/blob/blobtest"
+	"github.com/celestiaorg/celestia-node/libs/authtoken"
 	"github.com/celestiaorg/celestia-node/nodebuilder"
 	"github.com/celestiaorg/celestia-node/nodebuilder/node"
 	"github.com/celestiaorg/celestia-node/nodebuilder/tests/swamp"
 )
 
+func getAdminClient(ctx context.Context, nd *nodebuilder.Node, t *testing.T) *client.Client {
+	t.Helper()
+
+	signer := nd.AdminSigner
+	listenAddr := "ws://" + nd.RPCServer.ListenAddr()
+
+	jwt, err := authtoken.NewSignedJWT(signer, []auth.Permission{"public", "read", "write", "admin"})
+	require.NoError(t, err)
+
+	client, err := client.NewClient(ctx, listenAddr, jwt)
+	require.NoError(t, err)
+
+	return client
+}
+
```

Comment:
Eventually, we will abstract that away. E.g. `swamp.NewBridge() API` will give an API which is programmatic API or over RPC, depending on the node setup

Reply:
The current UX of the node as a library is not there yet.